src/_listObject.scss

```diff
@@ -0,0 +1,220 @@
+
+        @include fd-rtl() {
+          text-align: $fd-list-object-align-left;
+          padding-right: $fd-list-object-with-padding-value-one !important;
+          padding-left: $fd-list-object-with-value-zero !important;
+          margin-left: $fd-list-object-with-value-zero !important;
+        }
+      }
+
+      &--bottom-row {
+        display: flex;
+        align-items: flex-end;
+        justify-content: flex-end;
+
+        > div:only-child {
+          padding: $fd-list-object-with-value-zero;
+        }
+      }
+
+      &--bottom-left {
+        text-align: $fd-list-object-align-left;
+        padding:
+          $fd-list-object-with-padding-value-half
+          $fd-list-object-with-padding-value-half
+          $fd-list-object-with-padding-value-half
+          $fd-list-object-with-value-zero;
+
+        @include fd-rtl() {
+          text-align: $fd-list-object-align-right;
+          padding:
+            $fd-list-object-with-padding-value-half
+            $fd-list-object-with-value-zero
+            $fd-list-object-with-padding-value-half
+            $fd-list-object-with-padding-value-half;
+        }
+      }
+
+      &--bottom-right {
+        @include fd-ellipsis();
+
+        width: 50%;
+        text-align: $fd-list-object-align-right;
+        margin-right: $fd-list-object-with-value-zero;
+        padding:
+          $fd-list-object-with-padding-value-half
+          $fd-list-object-with-value-zero
+          $fd-list-object-with-padding-value-half
+          $fd-list-object-with-padding-value-half;
+
+        @include fd-rtl() {
+          text-align: $fd-list-object-align-left;
+          margin-left: $fd-list-object-with-value-zero;
+          padding:
+            $fd-list-object-with-padding-value-half
+            $fd-list-object-with-padding-value-half
+            $fd-list-object-with-padding-value-half
+            $fd-list-object-with-value-zero;
+        }
+      }
+
+      &--bottom {
+        margin-top: 0.25rem;
+      }
+    }
+
+    .#{$block}__object-number {
+      vertical-align: $fd-list-object-with-vertical-align-top !important;
+      margin-right: $fd-list-object-with-value-zero;
+
+      &-text {
+        font-weight: normal;
+        font-size: 1.375rem;
+        font-family: $fd-list-object-number-font-family;
+        text-shadow: $fd-list-object-with-no-text-shadow;
+      }
+
+      &-unit {
+        font-family: $fd-list-object-number-font-family;
+        font-size: $fd-list-object-font-size;
+        text-shadow: $fd-list-object-with-no-text-shadow;
+      }
+
+      &-text,
+      &-unit {
+        display: block !important;
+        padding-top: $fd-list-object-with-value-zero !important;
+      }
+
+      &-text::after {
+        content: none;
+      }
+
+      @include fd-rtl() {
+        margin-left: $fd-list-object-with-value-zero;
+      }
+    }
+
+    .#{$block}__object-identifier {
+      &-title {
+        @include fd-reset();
```

Comment:
Why does this need `fd-reset`? It's not a new element block. Please remove

Reply:
updated